README.md

```diff
@@ -0,0 +1 @@
+Demo READ.me
```

Comment:
This is much better and it seems you have the basic understanding of the git working as well, so that's grand. I am going to approve the MR and you can merge this.